### PR TITLE
#1-4Task. Alex.Mikhaltsev

### PR DESCRIPTION
**Pull request by Aleksan4e3**
https://travis-ci.org/github/Aleksan4e3/js-assignments/builds/704296349

### CODE REVIEW

File: task/01-strings-tasks.js [hunk 1]
```diff
@@ -69,7 +69,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    return value.substring(7, value.length - 1);
 }
```
Comment (IlyaMokin) [inline task/01-strings-tasks.js:73]: looks good, but please note substring can work with negative numbers also to take symbols from end. In the case `substring(7, -1);` can do the same.

File: task/04-arrays-tasks.js [hunk 2]
```diff
@@ -415,7 +424,10 @@ function toStringList(arr) {
  *      { country: 'Russia',  city: 'Saint Petersburg' }
  */
 function sortCitiesArray(arr) {
-   throw new Error('Not implemented');
+   return arr.sort((a, b) =>
+      a.country.localeCompare(b.country) === 0 ?
+         a.city.localeCompare(b.city) :
+         a.country.localeCompare(b.country));
 }
```
Comment (IlyaMokin) [inline task/04-arrays-tasks.js:431]: !

**Comment by IlyaMokin** [comment]
Closing the PR as accepted

**Comment by Aleksan4e3** [comment]
> Looks good!
> One note, the PR has redundant changes in styling, spaces, comments....
> Please follow the simple rules in the future:
> 
> * do not add any non logic changes to code if review is required
> * if the changes in styling are important - you can do it as separate PR with styling only changes to avoid noise in the main PR

It happens because I use hot key 'Alt+Shift+F' for alignment my code. I won`t doing so more starting 7th task as I already completed task5 and almost task6

**Pull request closed without merging**